core/conn/jdbcT4/pom.xml

```diff
@@ -27,11 +27,54 @@
 -->
 
   <modelVersion>4.0.0</modelVersion>
-  <groupId>org.trafodion.jdbc.t4</groupId>
+  <!--<parent>
+    <groupId>org.apache.trafodion</groupId>
```

Comment:
Package name for jdbcT4  is org.trafodion.jdbc.t4. Just changing the group id to contain apache in the package name without the package structure change in jdbcT4 code will not make it work.